pkg/weather-client/client.go

```diff
@@ -83,10 +28,37 @@ func New() WeatherClient {
 	return newNoaaClient()
 }
 
-func (nc *NoaaClient) GetForecast(xCoordinate float32, yCoordinate float32) (*ForecastResponse, error) {
-	// BOU weather office(boulder), 52X, 75Y this is boulder
-	url := fmt.Sprintf("%s/gridpoints/BOU/%f,%f/forecast/hourly",
-		nc.APIEndpoint, xCoordinate, yCoordinate)
+func (nc *NoaaClient) getNoaaPointInfo(lat float64, lng float64) (xCoord float64, yCoord float64, stationCode string, err error) {
```

Comment:
**nit:** Put private methods/functions at the bottom of files so exported (public) ones show up first.